src/loaders/HubsTextureLoader.js

```diff
@@ -76,13 +96,15 @@ export default class HubsTextureLoader {
       } else if (colorType === 0 || colorType === 2) {
         transparent = false;
       }
-
-      url = URL.createObjectURL(blob);
-    } else if (finalContentType === "image/jpeg") {
+    } else if (signaturesEqual(JPEGFileSignature, bytes)) {
       // JPEGs can't have an alpha channel, so memory can be saved by storing them as RGB.
       transparent = false;
+    } else {
+      console.warn(`Couldn't detect image type for: "${src}". Using RGBA pixel format.`);
     }
 
+    url = URL.createObjectURL(blob);
```

Comment:
I think we only need to bother with this in the case we have to fallback to Image. We already have the data here we need to create an `ImageBitmap`, so I don't think we need to go through the trouble of creating, fetching, and revoking this object url only to convert it back to an TypedArray to then create an ImageBitmap.

Reply:
Unfortunately there isn't a good way of doing that 😞 at least not that I can find. So I guess we'll stick with this for now.